srcpkgs/github-cli/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'github-cli'
 pkgname=github-cli
-version=2.0.0
+version=2.2.0
 revision=1
 wrksrc="cli-${version}"
 build_style=go
 build_helper=qemu
 go_import_path="github.com/cli/cli/cmd/gh"
 go_ldflags="-X github.com/cli/cli/internal/build.Version=v${version}"
+go_mod_mode="off"
```

Comment:
Not using Go modules is definitely a regression. Can you show where it fails?

Reply:
TBH: I don't know where execatly the problem is, but I get this error during building:

```shell
=> github-cli-2.2.0_1: building [go] [qemu] for x86_64...
   [host] go-1.17.2_1: found (https://alpha.de.repo.voidlinux.org/current)
=> github-cli-2.2.0_1: installing host dependencies: go-1.17.2_1 ...
=> github-cli-2.2.0_1: running pre-build hook: 02-script-wrapper ...
=> github-cli-2.2.0_1: running pre_build ...
=> github-cli-2.2.0_1: running do_build ...
no required module provides package github.com/cli/cli/cmd/gh; to add it:
	go get github.com/cli/cli/cmd/gh
=> ERROR: github-cli-2.2.0_1: do_build: 'go install -p "$XBPS_MAKEJOBS" -mod="${go_mod_mode}" -x -tags "${go_build_tags}" -ldflags "${go_ldflags}" ${go_package}' exited with 1
=> ERROR:   in do_build() at common/build-style/go.sh:36
```

After I added `go_mod_mode=off` it worked, so I pushed it.